core/src/main/java/org/jdbi/v3/core/Jdbi.java

```diff
@@ -62,6 +62,15 @@ private Jdbi(ConnectionFactory connectionFactory) {
         this.connectionFactory = connectionFactory;
     }
 
+    /**
+     * @param connection db connection
+     *
+     * @return a Jdbi which works on single connection
+     */
+    public static Jdbi create(Connection connection) {
```

Comment:
Let's `@Beta` this until we have some appropriate disclaimers in place. I'm thinking of people using this method naively assuming it works the same as a connection pool.